### PR TITLE
Gruntfile.js optimize, SassyCast repo fixed & .gitignore fixed

### PR DESCRIPTION
**Pull request by lcalvino**
Fixes on:

-Gruntfile.js
-bower.json
-.gitignore

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -41,6 +41,14 @@ module.exports = function (grunt) {
           livereload: true,
           debug: true
         }
+      },
+      doc: {
+        options: {
+          port: 1338,
```
Comment (fcoronelmakingsense) [inline Gruntfile.js]: can change to <code>config.server_doc_port</code> ?

**Comment by arzafran** [comment]
Looks good 2 me.
keep up with the good work!

:+1:

**Comment by fcoronelmakingsense** [comment]
:+1: Nice work!

File: Gruntfile.js (hunk 1 above)
Comment (lcalvino) [inline Gruntfile.js]: Fixed, thanks @fcoronelmakingsense !

**Pull request merged**